src/darwin/Framework/CHIP/MTRAttributeCacheContainer.mm

```diff
@@ -99,12 +98,12 @@ - (void)readAttributeWithEndpointId:(NSNumber * _Nullable)endpointId
             return;
         }
         __auto_type controllerId = self.xpcControllerId;
-        uint64_t nodeId = self.deviceId;
+        NSNumber * nodeId = self.deviceId;
         [xpcConnection
             getProxyHandleWithCompletion:^(dispatch_queue_t _Nonnull queue, MTRDeviceControllerXPCProxyHandle * _Nullable handle) {
                 if (handle) {
                     [handle.proxy readAttributeCacheWithController:controllerId
-                                                            nodeId:nodeId
+                                                            nodeId:[nodeId unsignedLongLongValue]
```

Comment:
I can do the dot bit, yes.

Fot the spelling, I've been matching function arg spelling to the selector, and the selector here can't change so far because I have not figured out how to change it compatibly.

Reply:
Did the dot thing, except in code where clearly the other convention is in use.